lib/settingslogic.rb

```diff
@@ -100,7 +100,7 @@ def initialize(hash_or_file = self.class.source, section = nil)
       self.replace hash_or_file
     else
       file_contents = open(hash_or_file).read
-      hash = file_contents.empty? ? {} : YAML.load(ERB.new(file_contents).result).to_hash
+      hash = file_contents.empty? ? {} : YAML.safe_load(ERB.new(file_contents).result, aliases: true).to_hash
```

Comment:
We don't have the same need to over complicate this. This argument is supported back to ruby 2.5 and psych >3 from what I can tell.  Since a user would need to pick the new branch and version at this time, we don't need to protect against older setups.